nixos/modules/services/backup/restic.nix

```diff
@@ -402,6 +413,7 @@ in
               EnvironmentFile = backup.environmentFile;
             };
         }
+        // lib.optionalAttrs (backup.unitConfig != null) backup.unitConfig
```

Comment:
This shouldn't be a `//` update. The issue here is that it doesn't actually merge the values given above with the ones specified in `unitConfig`. For example, if I give `unitConfig.serviceConfig = {};`, it removes all the config given in this module.

I know you can use `updateRecursive` or `mkMerge` here.

Reply:
`mkMerge` did the trick -- thanks!